zerver/lib/test_classes.py

```diff
@@ -1692,10 +1692,10 @@ def subscribe_realm_to_monthly_plan_on_manual_license_management(
         )
 
     @contextmanager
-    def tornado_redirected_to_list(
-        self, lst: List[Mapping[str, Any]], expected_num_events: int
-    ) -> Iterator[None]:
-        lst.clear()
+    def capture_send_event_calls(
+        self, expected_num_events: int
```

Comment:
I believe [the proposal](https://chat.zulip.org/#narrow/stream/3-backend/topic/tornado_redirected_to_list/near/1540990) was to rename this argument `expected_events`.

Reply:
After [discussion](https://chat.zulip.org/#narrow/stream/3-backend/topic/tornado_redirected_to_list/near/1541855), we are planning to keep using `expected_num_events`